src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -235,17 +242,24 @@ private static string GetSymbolsPath(string packagePath, bool isSnupkg)
                 Path.GetFileName(packageToPush),
                 sourceName));
 
+            bool showPushCommandPackagePushed = true;
+
             if (sourceUri.IsFile)
             {
-                await PushPackageToFileSystem(sourceUri, packageToPush, log, token);
+                await PushPackageToFileSystem(sourceUri, packageToPush, skipDuplicate, log, token);
             }
             else
             {
                 var length = new FileInfo(packageToPush).Length;
-                await PushPackageToServer(source, apiKey, packageToPush, length, noServiceEndpoint, requestTimeout, log, token);
+                showPushCommandPackagePushed = await PushPackageToServer(source, apiKey, packageToPush, length, noServiceEndpoint, skipDuplicate
```

Comment:
nit: formatting

Reply:
Can you please elaborate?

Reply:
Ideally you have 1 param per line or all on 1 line. 

Specifically here, though, I was referring to the line below
> , requestTimeout, log, tokenSource.Token, isSnupkgPush: false);
should be
> requestTimeout, log, tokenSource.Token, isSnupkgPush: false);

The comma `,` goes on the previous line.